doc/build/core/temporal.rst

```diff
@@ -0,0 +1,238 @@
+.. _metadata_temporal:
```

Comment:
**mike bayer** (zzzeek) wrote:

this is a lot of effort, great work.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3803

Reply:
**Trevor Gross** (tgross35) wrote:

Thanks!

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3803